arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,903 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1",
```

Comment:
There can only be one version field in the info section and it's value has to be the api-version value.